addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -431,6 +431,91 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     );
 });
 
+QUnit.test('message list for tracking field', async function (assert) {
+    assert.expect(3);
+    await this.start();
+    this.data["mail.message"].fields.tracking_value_ids = {
+        relation: "mail.tracking.values",
+        string: "Tracking Values",
+        type: "many2many",
+    }
+    const thread = this.env.models['mail.thread'].create({
+        channel_type: 'channel',
+        id: 100,
+        members: [['insert', [
+            {
+                email: "john@example.com",
+                id: 9,
```

Comment:
prefer ID value above 10 to avoid conflicts with potential "default" data from server (this is purely arbitrary)

note that in this case you can remove this. Just put thread in this.data before start, and current partner will be member by default which will be enough for the test.